.github/workflows/test.yml

```
@@ -2,6 +2,9 @@ name: tests
 
 on:
   push:
+    branches:
+      - master
+  pull_request:
```

```suggestion
  pull_request:
    types: [opened, reopened, synchronize]
```
Should limit the PR events that trigger this